tools/alpine/packages

```diff
@@ -0,0 +1,162 @@
+alpine-baselayout
```

Comment:
How is this generated? It looks like it is the some of all the packages used in the repository *plus* their dependencies. What are the runes I have type if I want to add another package? Can we add a README in `./tools/alpine` to describe the process?